xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -1234,7 +1238,7 @@ struct AppModel: ModelProtocol {
                 environment: environment
             )
         case .succeedMoveEntry, .succeedMergeEntry, .succeedLogActivity, .succeedUpdateAudience,
-                .succeedAssignNoteColor, .succeedAppendToEntry:
+                .succeedAssignNoteColor, .succeedAppendToEntry, .succeedUpdateLikeStatus:
```

Comment:
Nit: I generally prefer to break out cases like this into separate arms because I find it easier for my eyeball parsers to scan.